server.js

```diff
@@ -63,11 +63,20 @@ function songCommand(target, context, params) {
     client.say(target, 'Unable to fetch current song.').catch(e => console.log);
     return;
   }
-  const message = `cmgriffing is currently listening to ${song.songName} by ${
-    song.artist
-  }. It is track #${song.songNumber} on ${
-    song.albumName
-  }. You can find the album here: ${song.albumURL}`;
+  let message;
+  if (song.songNumber) {
```

Comment:
Could we refactor this into more of a "string builder" in which each sentence is conditionally added to the message?